Fix build after [252515@main](https://commits.webkit.org/252515@main)

danae404 opened the pull request:
#### 7c9eacf8ae34b243f5e09a8d97696faf3ef368b5
<pre>
Fix build after [252515@main](<a href="https://commits.webkit.org/252515@main)">https://commits.webkit.org/252515@main)</a>
<a href="https://bugs.webkit.org/show_bug.cgi?id=242823">https://bugs.webkit.org/show_bug.cgi?id=242823</a>
rdar://97098003

Reviewed by Nikolas Zimmermann.

* Tools/TestWebKitAPI/Tests/WebKitCocoa/FindInPageAPI.mm:
Imported &lt;WebKit/WKPreferencesPrivate.h&gt;

Canonical link: <a href="https://commits.webkit.org/252525@main">https://commits.webkit.org/252525@main</a>
</pre>

nikolaszimmermann reviewed (approved):
Thanks for the quick fix. Let's land with unsafe merge-queue to unbreak the world :-)

danae404 committed:
Fix build after [252515@main](https://commits.webkit.org/252515@main)
https://bugs.webkit.org/show_bug.cgi?id=242823
rdar://97098003

Reviewed by Nikolas Zimmermann.

* Tools/TestWebKitAPI/Tests/WebKitCocoa/FindInPageAPI.mm:
Imported <WebKit/WKPreferencesPrivate.h>

Canonical link: https://commits.webkit.org/252525@main
